store/sdk/src/test/java/org/apache/carbondata/sdk/file/TestUtil.java

```diff
@@ -17,20 +17,58 @@
 
 package org.apache.carbondata.sdk.file;
 
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.DataInputStream;
 import java.io.File;
 import java.io.FileFilter;
 import java.io.IOException;
+import java.io.InputStream;
 
 import org.apache.carbondata.common.exceptions.sql.InvalidLoadOptionException;
 import org.apache.carbondata.core.constants.CarbonCommonConstants;
 import org.apache.carbondata.core.datastore.impl.FileFactory;
 import org.apache.carbondata.core.util.CarbonProperties;
 import org.apache.carbondata.core.util.path.CarbonTablePath;
 
+import org.apache.avro.file.DataFileWriter;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericDatumReader;
+import org.apache.avro.generic.GenericDatumWriter;
+import org.apache.avro.io.DecoderFactory;
+import org.apache.avro.io.Encoder;
+import org.apache.avro.io.JsonDecoder;
 import org.junit.Assert;
 
 public class TestUtil {
 
+  public static GenericData.Record jsonToAvro(String json, String avroSchema) throws IOException {
+    InputStream input = null;
+    DataFileWriter writer = null;
+    Encoder encoder = null;
+    ByteArrayOutputStream output = null;
+    try {
```

Comment:
This is the 3rd place same code. same comment as above. Move to core module test util file and try to reusue same.

Reply:
test-cases are from two different packages , so we should write the util class separately.